.github/workflows/main.yml

```diff
@@ -13,17 +14,29 @@ jobs:
       uses: actions/checkout@v2
 
     - name: Build PostgreSQL container image
-      run: docker build --build-arg DATABASE_TYPE=postgresql --tag docker.pkg.github.com/mikecao/umami/umami:postgresql-$(echo $GITHUB_SHA | head -c7) .
+      run: | 
+        docker build --build-arg DATABASE_TYPE=postgresql \
+        --tag ghcr.io/$GITHUB_ACTOR/umami:postgresql-$(echo $GITHUB_SHA | head -c7) \
```

Comment:
`$GITHUB_ACTOR` is the GitHub account which is running the workflow, in this case it'll be `mikecao`, but it means that if anyone forks it will use their account and they'll be able to push their own images out of the box.